extlinks/common/helpers.py

```diff
@@ -241,13 +241,22 @@ def filter_queryset(queryset, filter_dict):
                 on_user_list=True
             )
 
+
     if 'namespace_id' in filter_dict:
         namespace_id = filter_dict['namespace_id']
         if namespace_id is not None:
             queryset = queryset.filter(
                 page_namespace=namespace_id
             )
 
+
+    if 'bot_edits' in filter_dict:
+        bot_edits = filter_dict['bot_edits']
+        if bot_edits:
+            queryset = queryset.filter(
+                bot_edits=True
```

Comment:
This is the cause of the current error. You're filtering the links queryset on the field `bot_edits`, which doesn't exist. If you head over to take a look at the LinkEvent model, you'll see the field which flags whether an event was made by a bot is called `user_is_bot`: https://github.com/WikipediaLibrary/externallinks/blob/master/extlinks/links/models.py#L68

Additionally, I think we should reverse the logic here. By default, all edits are included in results. We should change the button to be "Exclude bots", and use `queryset.exclude(user_is_bot=True)` if selected. (We could do the reverse and have the box ticked by default, but that strikes me as more complicated to implement, so this is fine).

In addition to updating the label next to the button, we should also update the form name from `bot_edits` to `exclude_bots` for clarity.